zone_marking/Lines lines
---
std_msgs/Bool outcome